python/GafferImageTest/ImageWriterTest.py

```diff
@@ -126,7 +126,7 @@ def testTiffWrite( self ) :
 				{ 'value': "float", 'metadata': { 'oiio:BitsPerSample': IECore.IntData( 32 ) } },
 			]
 
-		self.__testExtension( "tif", "tiff", options = options, metadataToIgnore = [ "tiff:RowsPerStrip" ] )
+		self.__testExtension( "tif", "tiff", options = options, metadataToIgnore = [ "tiff:RowsPerStrip", "IPTC:Creator" ] )
```

Comment:
it was failing when run locally here at IE. It seems reasonable to ignore this metadata